awx_collection/plugins/modules/export.py

```diff
@@ -111,24 +115,12 @@
 def main():
     argument_spec = dict(
         all=dict(type='bool', default=False),
-        credential_types=dict(type='str'),
```

Comment:
@john-westcott-iv ditto for these, it worries me to completely remove parts of something called `argument_spec`.

Reply:
As long as it shows up in `EXPORTABLE_RESOURCES` in awxkit these are ok to remove here because of lines 129-131.